libcodechecker/output_formatters.py

```diff
@@ -0,0 +1,186 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+Contains functions to format and pretty-print data from two-dimensional arrays.
+"""
+
+import json
+
+# The supported formats the users should specify. (This is not an exhaustive
+# list of ALL formats available.)
+USER_FORMATS = ['rows', 'table', 'csv', 'json']
```

Comment:
Right now it could be that there isn't.

Once this PR is merged, the upcoming `checkers` and `analyzers` command will use this as their output format `choices`.